option.go

```diff
@@ -35,16 +35,19 @@ var (
 		WithPolicyErrRetryInterval("1m"),
 		WithPubkeyErrRetryInterval("1m"),
 		WithJwkErrRetryInterval("1m"),
-		WithATProcessorParams(NewATProcessorParam(true, "1h", "1h")),
+		WithATProcessorParam(NewATProcessorParam(true, true, "1h", "1h", false, nil)),
 		WithRTVerifyRoleToken(true),
 		WithRCVerifyRoleCert(true),
 	}
 )
 
 type ATProcessorParam struct {
-	verifyCertThumbprint bool
-	certBackdateDur      string
-	certOffsetDur        string
+	verifyOAuth2AccessToken bool
+	verifyCertThumbprint    bool
+	certBackdateDur         string
+	certOffsetDur           string
+	verifyTokenClientID     bool
```

Comment:
```suggestion
	verifyClientID     bool
```

Reply:
I think that `verify"Token"ClientID` corresponds to `verify"Cert"Thumbprint`, but is there something wrong with it?

Reply:
Since it's the part that verifies the elements of the token, I'll modify it as suggested.

Reply:
7bc9d06